### ISSUE TITLE
Export-Excel: Spaces not allowed in TableName

### ISSUE THREAD

**Issue by DarkLite1** [dfinke/ImportExcel#165]
Consider the following code:

```
$ExcelFile = $env:TEMP + '\test.xlsx'
Remove-Item $ExcelFile -EA Ignore

$ExcelParams = @{
    Path          = $ExcelFile
    AutoSize      = $true
    FreezeTopRow  = $true
    TableName     = "Tablename with spaces"
    WorkSheetname = "Test sheet"
}
Get-Process | Export-Excel @ExcelParams
```

This outputs the following errors:
```
Exception calling "Add" with "2" argument(s): "Tablename has spaces"
At C:\Program Files\WindowsPowerShell\Modules\ImportExcel\Export-Excel.ps1:231 char:13
+             $tbl = $ws.Tables.Add($targetRange, $TableName)
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : ArgumentException
 
The property 'TableStyle' cannot be found on this object. Verify that the property exists and can be set.
At C:\Program Files\WindowsPowerShell\Modules\ImportExcel\Export-Excel.ps1:233 char:13
+             $tbl.TableStyle=$TableStyle
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : PropertyNotFound
```

When using the previous EPPLus.dll this wasn't a problem. But recently I updated the EPPlus.dll with the new one from the master branch and this problem pops up. The spaces in TableName is not a problem in the PowerShell code but more in the EPPlus.dll library I guess. 

Thank you for checking this.

**Comment by DarkLite1** [comment]
This is an exception thrown by the EPPLus library. see line [120 here](https://github.com/DHMechatronicAG/EPPlus/blob/master/EPPlus/Table/ExcelTableCollection.cs).:

        private void ValidateTableName(string Name)
        {
            if (string.IsNullOrEmpty(Name))
            {
                throw new ArgumentException("Tablename is null or empty");
            }

            char firstLetterOfName = Name[0];
            if (Char.IsLetter(firstLetterOfName) == false && firstLetterOfName != '_' && firstLetterOfName != '\\')
            {
                throw new ArgumentException("Tablename start with invalid character");
            }

            if (Name.Contains(" "))
            {
                throw new ArgumentException("Tablename has spaces");
            }

        }

The strange thing is with the previous dll they didn't throw this error. Opened a [related issue](https://github.com/DHMechatronicAG/EPPlus/issues/1) on their GitHub to ask for more information about this change.

**Comment by dfinke** [comment]
I think handling and throwing an error is best. Changing the name of the table from what the user expected will be confusing.

**Comment by dfinke** [comment]
Probably something like

```
if($TableName -and $TableName.Contains(" ")) {
    throw "TableName [$($TableName)] is not valid, spaces not allowed."
}
```

**Comment by DarkLite1** [comment]
Hmmmm... I think it would be better to put that in the parameter validation part. Check my PR, it should solve these issues. I've tried to mimic the original error checking as much as possible.

**Comment by DarkLite1** [comment]
Fixed with PR #165
